firefox-ios/Shared/Strings.swift

```diff
@@ -989,6 +989,24 @@ extension String {
                 comment: "A button at bottom of the Firefox homepage that, when clicked, takes users straight to the settings options, where they can customize the Firefox Home page")
         }
 
+        public struct PrivacyNotice {
+            public static let Body = MZLocalizedString(
+                key: "FirefoxHome.PrivacyNotice.Body.v148",
+                tableName: "FirefoxHomepage",
+                value: "We’ve updated our %@ to reflect the latest features in Firefox. %@",
```

Comment:
We shouldn't have the app name in strings. This should be a placeholder as well. :)